src/Xrefcheck/CLI.hs

```diff
@@ -185,9 +185,16 @@ optionsParser = do
         help "Do not display progress bar during verification."
     , pure Nothing
     ]
-  oColorMode <- flag WithColors WithoutColors $
-    long "no-color" <>
-    help "Disable ANSI coloring of output."
+  oColorMode <- asum
+    [ flag' (Just WithColors) $
+        long "color" <>
+        help "Enable ANSI coloring of output. \
+            \This is enabled by default when `CI` env var is set to true."
```

Comment:
Very good that you included this clarification. However I read it like: "it is enabled if and only if `CI` env var is set to true"  :thinking: 

Maybe say it differently, like "When `CI` env var is set to true, this option will be enabled by default". I only swapped the sub-sentences, but to me it seems like avoiding the ambiguity.

I may be wrong about all this though.

Reply:
@Martoon-00 I have clarified the option description, and also included some tests regarding the new behavior in CI.